.github/workflows/infra-review-p-itn.yml

```diff
@@ -0,0 +1,25 @@
+name: PR Infrastructure Prod Italy North
```

Comment:
can we change these titles with the intent of these workflow? ie. something alike
"Run Terraform appy on..." "Run Terraform plan on..." 

it's not clear to me why we have 4 workflows for infra